syntax = "proto3";

package logging;

service LoggingService {
  rpc StreamLogs (stream LogMessage) returns (LogResponse);
}

message LogMessage {
  string log = 1; // The log message content
  string service = 2; // Service sending the log (e.g., API Service, DB Service)
  int64 timestamp = 3; // Timestamp of the log
}

message LogResponse {
  string status = 1; // Status of the log reception (e.g., "SUCCESS")
}

//python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. logging_service.proto
